Server/Backend/README.md

```diff
@@ -39,6 +39,50 @@ Metadata to identify and retrieve files are stored in the relational database, w
 --------------------------------------------------------------------------------
 ## Dependencies
 
+The backend itself manages its dependencies using Maven.
+However, there are additional dependencies to run the complete stack effectively.
+
+### Maven
+
+You can use the following command to check for outdated versions of direct dependencies:
+
+```sh
+./mvnw versions:display-dependency-updates
+```
+
+Please note that updating dependencies might require code changes.
+It is usually safe -- and strongly suggested -- to bump patch level versions.
+This is typically the third part of the version number, e.g., `1.2.3` -> `1.2.5`.
+Changes to the first two segments are often associated to breaking changes and need additional manual review and testing.
```

Comment:
This should only be true for the first segment, right? Or are the dependencies using something other than semantic versioning?

Reply:
We cannot guarantee that dependencies use semantic versioning.